aspnet-core/Chart/Appearance.md

```diff
@@ -24,7 +24,7 @@ The chart displays different series in different colors, by default. You can cus
 {% endhighlight %}
 
 
-![](Appearance_images/Appearance_img1.png)
+![Custom color palette in APS.NET Core Chart](Appearance_images/aspnetcore-chart-appearance.png)
```

Comment:
Change like this - Appearance in APS.NET Core Chart

Reply:
done